### PR TITLE
PR: Add basic requirements in the template.

### PR DESCRIPTION
**Pull request by rlaverde**

### CODE REVIEW

File: {{ cookiecutter.repo_name }}/requirements.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+spyder==3.1.3
```
Comment (goanpeca) [inline {{ cookiecutter.repo_name }}/requirements.txt]: Equal?

**Review by goanpeca — CHANGES_REQUESTED** [review]

File: {{ cookiecutter.repo_name }}/requirements.txt (hunk 1 above)
Comment (rlaverde) [reply {{ cookiecutter.repo_name }}/requirements.txt]: yes, I's the latest stable version, and I want to update dependencies using pyup #27 so dependencies should be pinned

Comment (goanpeca) [reply {{ cookiecutter.repo_name }}/requirements.txt]: But why not >=, I don't like exact pinning, thats a bad practice

Comment (rlaverde) [reply {{ cookiecutter.repo_name }}/requirements.txt]: But dependencies will updated using pyup, and exact pinning avoid them to be automatically updated.

Comment (rlaverde) [reply {{ cookiecutter.repo_name }}/requirements.txt]: I'll change this for >= and exclude template dependencies from pyup

**Review by goanpeca — APPROVED** [review]
maybe just 3.1 is good enough?

**Pull request merged**